DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: rticles
 Type: Package
 Title: Article Formats for R Markdown
-Version: 0.5.2
+Version: 0.5.2.9001
```

Comment:
We will bump version after merging the PR. You don't have to change the version number here.

Reply:
okay